extensions/binding/org.eclipse.smarthome.binding.bluetooth.bluegiga/NOTICE

```diff
@@ -12,6 +12,11 @@ https://www.eclipse.org/legal/epl-2.0/.
 
 https://github.com/eclipse/smarthome
 
+This project copies classes from the Z-Smart Systems Bluegiga Java Library,
```

Comment:
"copies" sounds like as if this is continuous. How about reformulating it to
"Some code of this bundle originates from the Z-Smart Systems Bluegiga Java Library, which can be found at"?